tests/db/migration_tests/test_v202508040_to_v202508270.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright 2025 Inmanta
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contact: code@inmanta.com
+"""
+
+import os
+import re
+from collections import abc
+
+import pytest
+
+file_name_regex = re.compile("test_v([0-9]{9})_to_v[0-9]{9}")
+part = file_name_regex.match(__name__)[1]
+
+
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), f"dumps/v{part}.sql"))
+async def test_drop_model_column(migrate_db_from: abc.Callable[[], abc.Awaitable[None]], postgresql_client) -> None:
+    """
+    Test the migration script that removes the 'model' column from the 'resource' table
+    """
+    # This migration only drops one column
+    await migrate_db_from()
```

Comment:
I wonder if we need to check of we can still read existing data using the new queries? or is this tested where we do the migration? 

(if we collapse the branches prior to going to master, do we collapse the DB updates as well?)